Chapters/Expressions/Expressions.pillar

```diff
@@ -506,15 +499,15 @@ EEAdditionTest >> testNegated
 	self assert: (EAddition new right: ep1; left: ep2) negated evaluate equals: -8
 ]]]
 
-The tools in Pharo support such a practice. If we tag a class method with the special annotation ==<sampleInstance>== the browser will show a little icon on the side and when we click on it, it will open an inspector on the new instance. 
+The tools in Pharo support such a practice. If we tag a class method with the special annotation ==<sampleInstance>== the browser will show a little icon on the side and when we click on it, it will open an inspector on the new instance:
 
 [[[
 EConstant class >> constant3
 	<sampleInstance>
 	^ self new value: 3
 ]]]
 
```

Comment:
We should add the code of constant5 because we use it latter.
Can you add it because I did not see it.
Tx

Reply:
done!